.github/workflows/ci.yml

```
@@ -32,3 +32,15 @@ jobs:
       run: |
         cargo run -- bisect-rustc --help | grep "Examples:"
         cargo run -- --help | grep "Commands:"
+
+  fmt:
+    name: rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Run rustfmt check
+        run: |
+          rustup component add rustfmt
+          cargo fmt --version
+          cargo fmt -- --check || (echo "Please reformat your code with 'cargo fmt' (version $(cargo fmt --version))"; false)
```

Since rustfmt has grown its own direct check flag since this line was originally written:
```suggestion
          cargo fmt --check || (echo "Please reformat your code with 'cargo fmt' (version $(cargo fmt --version))"; false)
```